parsl/executors/workqueue/executor.py

```diff
@@ -46,69 +46,71 @@ def WorkQueueSubmitThread(task_queue=multiprocessing.Queue(),
 
     logger.debug("Starting WorkQueue Submit/Wait Process")
 
-    orig_ppid = os.getppid()
-
-    wq_tasks = set()
-
-    continue_running = True
-
+    # Enable debugging flags and create logging file
     if wq_log_dir is not None:
-        wq_debug_log = os.path.join(wq_log_dir, "debug")
+        logger.debug("Setting debugging flags and creating logging file")
+        wq_debug_log = os.path.join(wq_log_dir, "debug_log")
         cctools_debug_flags_set("all")
         cctools_debug_config_file(wq_debug_log)
 
-    logger.debug("Creating Workqueue Object")
+    # Create WorkQueue queue object
+    logger.debug("Creating WorkQueue Object")
     try:
+        logger.debug("Listening on port {}".format(port))
         q = WorkQueue(port)
     except Exception as e:
-        logger.error("Unable to create Workqueue object: {}", format(e))
+        logger.error("Unable to create WorkQueue object: {}".format(e))
         raise e
 
+    # Specify WorkQueue queue attributes
     if project_name:
         q.specify_name(project_name)
-
     if project_password:
         q.specify_password(project_password)
     elif project_password_file:
         q.specify_password_file(project_password_file)
 
-    # Only write Logs when the log_dir is specified, which is most likely always will be
+    # Only write logs when the wq_log_dir is specified, which it most likely will be
```

Comment:
if that's a parsl-user-configured parameter, it's quite common to not want to specify an explicit log directory (or explicit anything, as much as you can get away with)

Reply:
The wq_log_dir is automatically specified according to the current directory. The only time it is would not be specified is if `os.path.join()` returns None, but otherwise the path is autoamatically created.